index.js

```diff
@@ -74,7 +73,7 @@ async function addPr(item, prs) {
         if (m == null)
             continue
         packages.add(m[1])
-        if (fileChange.status === "D")
+        if (fileChange.status === "D" || fileChange.status === "removed")
```

Comment:
It was generally working without upgrading Octokit except for this. But I thought fully upgrading Octokit was a good idea.